### ISSUE TITLE
Latest `private_chain` build has webpback problem with `randombytes` module?

### ISSUE THREAD

**Issue by roschler** [NeoNEXT/neoline#98]
I was able to build and load the previous release of the `private_chain` branch.  I just did a fresh pull of the `private_chain` branch and when I execute `yarn build`, I get the following error repeatedly in the build output:
```
    ERROR in ./extension/background/index.ts
    Module not found: Error: Can't resolve 'randomBytes' in '../neo/neoline/extension/background'
     @ ./extension/background/index.ts 99:18-40
```

I checked `package.json` and found this:
```
    "randombytes": "^2.1.0",
```

So I tried these steps next:

- yarn upgrade
- yarn add randombytes

These executed without error.  Then I tried `yarn build` again and got the same error, on both Windows 10 and UBuntu 14.04.  My guess is that there is problem with either the `webpack` build or the way `webpack` is resolving dependency graph for `randombytes` module given the latest `private_chain` build. Either way, I and perhaps others can not build the NeoLine N3 extension for Chrome because this problem is preventing the generation of a proper manifest file.

Here are additional warning messages from the build process that may be relevant:

```
WARNING in ./node_modules/@angular/compiler/src/util.js 10:24-31
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
    at CommonJsRequireContextDependency.getWarnings (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/dependencies/ContextDependency.js:40:18)
    at Compilation.reportDependencyErrorsAndWarnings (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compilation.js:1454:24)
    at /home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compilation.js:1258:10
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:42:1)
    at AsyncSeriesHook.lazyCompileHook (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.finish (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compilation.js:1253:28)
    at /home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compiler.js:672:17
    at _done (eval at create (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at eval (eval at create (/home/robert/Documents/GitHub/ME/neo/neoline/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:36:22)
    at /home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compilation.js:1185:12
    at /home/robert/Documents/GitHub/ME/neo/neoline/node_modules/@angular-devkit/build-angular/node_modules/webpack/lib/Compilation.js:1097:9
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
 @ ./src/app/popup/notification/deploy/deploy.component.ts
 @ ./src/app/popup/notification/notification.route.ts
 @ ./src/app/popup/notification/notification.module.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts

WARNING in /home/robert/Documents/GitHub/ME/neo/neoline/src/environments/environment.prod.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

WARNING in budgets: Exceeded maximum budget for initial-es2015. Budget 8 MB was not met by 2.21 MB with a total of 10.2 MB.

WARNING in budgets: Exceeded maximum budget for initial-es5. Budget 8 MB was not met by 2.65 MB with a total of 10.6 MB.
```

**Comment by roschler** [comment]
UPDATE: I downloaded an built the latest branch `fix_network_event`.  It does _not_ have this problem and I was able to build this version successfully.  Does this version have the fix for the TRANSACTION_CONFIRMED event not firing?

**Comment by hacfox** [comment]
> UPDATE: I downloaded an built the latest branch `fix_network_event`. It does _not_ have this problem and I was able to build this version successfully. Does this version have the fix for the TRANSACTION_CONFIRMED event not firing?

@roschler We tested it many times and none of these branches showed the problem. and one point: If the private node does not confirm, the transaction confirmation event will not be triggered. Are you sure the private node is working.

**Comment by roschler** [comment]
Hi @hacfox.  I am confused now.  On Discord/DM about a week ago, I told bozimeile that the TRANSACTION_CONFIRMED event was not firing for me.  He told me to do a fresh pull of `private_chain` and then rebuild and that would solve the problem.  I assumed that meant something was fixed in a newer build.  That is when I ran into the `randombytes` module problem.

_"Are you sure the private node is working?"_

Yes I know the node is working.  It is a local NEO Express instance running on a PC on my office intranet that I call from my Node.JS dev station.  The NeoLine N3 extension is able to communicate with it without problem.  But I am not receiving TRANSACTION_CONFIRMED event notifications after successfully submitting a transaction via the extension.

I'll go ahead and try the `fix_network_event` build in the meantime.  As I said, I can't build the `private_chain` branch currently because of the `randombytes` module resolution problem.  The `fix_network_event` branch does not have the `randombytes` issue.
